server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CICSLexer.g4

```diff
@@ -14,898 +14,5 @@
 
 lexer grammar CICSLexer;
```

Comment:
Why is this file left empty?

Reply:
There is story "Redesign Grammar", so we will extract CICS only words here, currently we took all under CommonLexer.